crates/recursor/src/recursor.rs

```diff
@@ -462,3 +485,107 @@ fn recursor_opts() -> ResolverOpts {
 
     options
 }
+
+//
```

Comment:
Even though this is a private method, it docs can be generated with `--document-private-items`. So it would be nice to use `///` here for the comment, and probably worth adding a link to the docs with `[RFC 8499](https://datatracker.ietf.org/doc/html/rfc8499)`

Reply:
Done